### ISSUE TITLE
Panels Tab Expandability

### ISSUE THREAD

**Issue by weiquu** [TEAMMATES/teammates#12534]
Currently, panels are unable to be expanded by tabbing to it and pressing spacebar or enter/return (at least, I'm unable to find a way to do so). The root issue seems to be that they are not captured by the tab order. Not sure if we should try to implement a fix for this to make the site more accessible

**Comment by jasonqiu212** [comment]
Hi! I believe #12567 should fix this issue. 

In #12567, each chevron is converted into a button. Thus, the chevron is captured by the tab order. Users can expand panels by tabbing to the chevron and pressing spacebar/enter. It also preserves the original functionality where clicking on the whole panel toggles the dropdown.

The only edge case is if expandable panels do not have this chevron.

**Comment by anjum-edge** [comment]
Add new features in chevrons because the tab order content can be shown once to show it single to multiple times. Original functionality can be changed because the changes in the code can be done and visible online so make sure to take care of that.
